org.adempiere.base.process/src/org/compiere/process/M_PriceList_Create.java

```diff
@@ -336,7 +339,7 @@ else if (M_Pricelist_Version_Base_ID == 0) {
 			/** Calculations	**/
 			MProductPrice[] pp = m_plv.getProductPrice(
 					"AND EXISTS (SELECT * FROM T_Selection s "
-							+ "WHERE s.T_Selection_ID=M_ProductPrice.M_Product_ID)");
+							+ "WHERE s.AD_PInstance_ID = " + m_AD_PInstance_ID + " AND s.T_Selection_ID=M_ProductPrice.M_Product_ID)");

```

Comment:
This one looks correct, like fixing a bug, I think is better to fix bugs in independent ticket - as is possible that the improvement is delayed, but the bug needs to be fixed a.s.a.p.

Reply:
Hi @globalqss 
I had done so if I had detected it earlier but I found it while testing and wondering why prices where changed which were not part of the selection. If adoption of this pull request really takes longer, I'll separate it.

Reply:
Yes - I think it would be good to separate it - bugs are treated different than improvements

Reply:
Ok, I'll create a separate ticket and pull request. I'll leave this one unchanged, as it doesn't change anything as soon as the bug is corrected.

Reply:
as IDEMPIERE-4469 created and pull request prepared

Reply:
Hi @CarlosRuiz-globalqss 
I thought that after committing IDEMPIERE-4469 this difference would disappear. Do I have to regenerate the pull request to get the changes taken into account?